static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -37,9 +39,11 @@ define([
         }, {
             id: 'variantA',
             test: function () {
-                //TODO check also if not a subscriber
                 detect.getFfOrGenericAdbockInstalled.then(function (adblockUsed) {
-                    if (adblockUsed && !config.page.isFront && !userFeatures.isPayingMember()) {
+                    if (adblockUsed && !config.page.isFront &&
+                        !userFeatures.isPayingMember() &&
+                        config.page.webTitle !== 'Subscriber number form' &&
```

Comment:
I don't know either, but so long as we always control this parameter, it must presumably suffice for a ten-day trial.